service/storage_service.cc

```diff
@@ -3728,6 +3728,24 @@ future<> storage_service::node_ops_done(utils::UUID ops_uuid) {
 future<> storage_service::node_ops_abort(utils::UUID ops_uuid) {
     slogger.debug("node_ops_abort: ops_uuid={}", ops_uuid);
     auto permit = co_await seastar::get_units(_node_ops_abort_sem, 1);
+
+    if (!ops_uuid) {
+        for (auto& [uuid, meta] : _node_ops) {
+            co_await meta.abort();
+            auto as = meta.get_abort_source();
+            if (as && !as->abort_requested()) {
+                as->request_abort();
+            }
+        }
+
+        for (auto it = _node_ops.begin(); it != _node_ops.end(); it = _node_ops.erase(it)) {
+            node_ops_meta_data& meta = it->second;
+            co_await meta.stop();
+        }
```

Comment:
This is duplicating the stop code below.  We should extract it to a method or lambda to avoid accidental divergence.

Reply:
This path does 2 passes: first it aborts all repairs, then it awaits their stop() so they are slightly different.
In any case, this area will is being converted to use the task manager infrastructure in #11597 so I don't think it's worth the complication.

Reply:
The first half of the code is shared. But if there are further changes coming, then fine.